project3/coordinator/coordinator_resources/messenger.cpp

```diff
@@ -50,7 +50,7 @@ void Messenger::LogMessage(MessageLog::Message *msg) {
 bool Messenger::SendMissedMessages(
     const MessageLog::Timestamp& reconnection_time) {
   int msg_count = 0;
-  for (const MessageLog::Message& msg :
+  for (MessageLog::Message msg :
```

Comment:
I think one you fix the previous comment this can be a `const` reference again also.